.gitignore

```diff
@@ -29,3 +29,6 @@ node_modules
 
 # Optional npm cache directory
 .npm
+
+# Environment Variables
+.env
```

Comment:
This file seems fine except for the fact that it did not prevent the .env file from ending up in the repository.